servers/quarkus-server/src/test/java/org/projectnessie/server/ITMetrics.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.jaxrs.AbstractTestRest;
+import org.projectnessie.server.profiles.QuarkusTestProfileInmemory;
+
+@QuarkusTest
+@TestProfile(value = QuarkusTestProfileInmemory.class)
+public class ITMetrics extends AbstractTestRest {
+  // We need to extend the AbstractTestRest because all Nessie metrics are created lazily.
+  // They will appear in the `/q/metrics` endpoint only when some REST actions are executed.
+
+  // this test is executed after all tests from the AbstractTestRest
+  @Test
+  void smokeTestMetrics() {
+    // when
+    String body =
+        RestAssured.given()
+            .when()
+            .basePath("/q/metrics")
+            .get()
+            .then()
+            .statusCode(200)
+            .extract()
+            .asString();
+
+    // then
+    assertThat(body).contains("jvm_threads_live_threads");
```

Comment:
seems ok for now. We'll follow up with whatever other metrics we want to verify in a separate PR, so let's merge what we have

Reply:
Just open it as a draft, nobody will/should touch it then